lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -692,6 +693,37 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return out.join( ',' );
 	});
 
+	/**
+	* Takes an integer value and returns the data at that index.
```

Comment:
@aayush0325 Would you mind moving this so that methods are defined in alphabetical order? You'll want to move to L461.